src/org/ods/usecase/LeVADocumentUseCase.groovy

```diff
@@ -571,14 +571,16 @@ class LeVADocumentUseCase extends DocGenUseCase {
             def riskPriority = obtainEnum("RiskPriority", r.riskPriority)
             r.riskPriority = riskPriority ? riskPriority.value : "N/A"
 
+            r.riskPriorityNumber = r.riskPriorityNumber ?: "N/A"
+
             return r
         }
 
         def proposedMeasuresDesription = this.project.getRisks().collect { r ->
             (r.getResolvedTests().collect {
                 if (!it) throw new IllegalArgumentException("Error: test for requirement ${r.key} could not be obtained. Check if all of ${r.tests.join(", ")} exist in JIRA")
-                [key: it.key, name: it.name, type: "test", referencesRisk: r.key]
-            } + r.getResolvedMitigations().collect { [key: it?.key, name: it?.name, type: "mitigation", referencesRisk: r.key] })
+                [key: it.key, name: it.name, description: it.description, type: "test", referencesRisk: r.key]
+            } + r.getResolvedMitigations().collect { [key: it?.key, name: it?.name, description: it?.description, type: "mitigation", referencesRisk: r.key] })
```

Comment:
@martsec are you expecting `it` to be null here? i.e., have you seen cases where `getResolvedMitigations()` returns a list including nulls?

Reply:
Hmm... It might have been a consequence of old code behavior. I'm going to change it